Resources/Prototypes/Nyanotrasen/Entities/Objects/Consumable/Food/moth.yml

```diff
@@ -0,0 +1,1210 @@
+# Moth Foods, the three C's: cheese, coleslaw, and cotton
+# Base ingredients
+- type: entity
+  name: herby cheese
+  parent: FoodBakingBase
+  id: FoodMothHerbyCheese
+  description: As a staple of mothic cuisine, cheese is often augmented with various flavours to keep variety in their diet.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - cheesy
+      - leafy
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: herbycheese
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 10
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 7
+        - ReagentId: Protein
+          Quantity: 2
+        - ReagentId: Cream
+          Quantity: 1
+#Tastes like cheese, herbs
+
+- type: entity
+  name: mothic salad
+  parent: FoodBowlBase
+  id: FoodMothSaladBase
+  description: A basic salad of cabbage, red onion and tomato.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - cabbage
+      - onion
+      - tomato
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: mothsaladbase
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxvol: 6
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 2
+        - ReagentId: Vitamin
+          Quantity: 4
```

Comment:
Healthy food is one thing, but it should also be somewhat nutritious. I know it makes sense for salads, but when players catch on that they don't quench hunger, they might not bother making it.

Reply:
I think the original idea for it was to serve as a base for other salads so I kept it to a similar status. Will boost it to 4nutri 2vit

Reply:
Cool!